### PR TITLE
Add documentation for select platform for La Marzocco

### PR DESCRIPTION
**Pull request by zweckj**
## Proposed change

Add documentation for select platform for La Marzocco

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/108222
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/lamarzocco.markdown [hunk 1]
```diff
@@ -54,3 +54,10 @@ Host:
 | Total coffees made | Counter for total coffees made| all | - |
 | Total flushes made | Counter for total flushes done | all | - |
 | Shot timer | Time the current brew is running | all | Only available when the *Host* was set during component configuration. |
+
+## Selects
+
+| Select name | Description | Options | Available for machines | 
+|-------------|-------------| ------------------------| ---------------------- |
+| Prebrew/-infusion mode | Whether to use prebrew, preinfusion or not | Disabled, Prebrew, Preinfusion | Linea Micra, Linea Mini, GS3 AV |
```
Comment (c0ffeeca7) [inline source/_integrations/lamarzocco.markdown]:
```suggestion
| Prebrew/-infusion mode | Whether to use prebrew, preinfusion, or neither | Disabled, Prebrew, Preinfusion | Linea Micra, Linea Mini, GS3 AV |
```

**Review by c0ffeeca7 — COMMENTED** [review]
tweak

**Comment by c0ffeeca7** [comment]
This steady flow of PRs on the subject keeps my mind preoccupied with coffee. You've got me browsing their website now.

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @zweckj :+1:

**Comment by zweckj** [comment]
> This steady flow of PRs on the subject keeps my mind preoccupied with coffee. You've got me browsing their website now.

I find it hilarious that you are the one reviewing those PRs (that name) 😂

**Comment by c0ffeeca7** [comment]
Haha, yes. When I saw the label, I kind of jumped on it. I've already measured if their smallest model would fit...

**Pull request merged**